### ISSUE TITLE
Manually switching boiler

### ISSUE THREAD

**Issue by WollyWonka** [robertklep/nefit-easy-client#10]
Hi Robert,
first thanx for your work. I'm happily using your client with Domoticz.

However, I would like to have more control on the heating system. As I can close the loop of the floorheating, and using Zigbee thermostats for the radiators I'm able to control these radiator heating units as well. 

My question is, if it is possible (using your implementation) to switch on the boiler without increasing the Setpoint? 
Namely, the Setpoint is related to the living room temperature and fiddling with this parameter feels a bit messy. 

Thanx for considering.
Regards, Jochem

**Comment by robertklep** [comment]
I've never run across a feature like that, and I doubt it exists. Detaching the boiler state from the setpoint seems to me like a possible recipe for disaster :)

This client is used to control the thermostat, not the boiler. It sounds to me like you want to do the latter. Perhaps the [OpenTherm Gateway](https://www.nodo-shop.nl/nl/opentherm-gateway/188-opentherm-gateway.html) fits your purpose better?

**Comment by WollyWonka** [comment]
Thanks Robert for your reply! To be honest, I already doubted if this was possible. However, my preference was not adding OpenTherm equipment to avoid multiple captains steering the ship.

I figured out a work-around. Touching the Nefit Easy physically, only changes the setpoint in rounded of halve values (1 or 0.5). So I can apply fractions to the setpoint value in my script to distinct manual switching the boiler or by domoticz.

**Comment by WollyWonka** [comment]
Thanks Robert for your reply! To be honest, I already doubted if this was possible. However, my preference was not adding OpenTherm equipment to avoid multiple captains steering the ship.

I figured out a work-around. Touching the Nefit Easy physically, only changes the setpoint in rounded of halve values (1 or 0.5). So I can apply fractions to the setpoint value in my script to distinct manual switching the boiler or by domoticz.
